BUILD

```diff
@@ -1467,10 +1467,10 @@ grpc_cc_library(
 )
 
 grpc_cc_library(
-    name = "xds_channel_creds",
+    name = "channel_creds",
     language = "c++",
     public_hdrs = [
-        "src/core/lib/security/credentials/xds/xds_channel_creds.h",
+        "src/core/lib/security/credentials/channel_creds.h",
```

Comment:
Suggest renaming this file to `channel_creds_registry.h`.

Reply:
Done.